docs/userguide/conversion/data.md

```diff
@@ -354,9 +347,11 @@ Each row has 256 entries; one for each possible byte value.
 The state table lines in the .ucm header conform to the following Extended
 Backus-Naur Form (EBNF)-like grammar (whitespace is allowed between all tokens):
 
+````
 row=\[\[firstentry ','\] entry (',' entry)\*\]
```

Comment:
Please remove the backslashes on this line

Reply:
Done